src/main/java/domain/AttendanceHistory.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.time.LocalDateTime;
+
+public class AttendanceHistory implements Comparable<AttendanceHistory> {
+    public final static int ABSENT_DEFAULT_HOUR = 23;
+    public final static int ABSENT_DEFAULT_MINUTE = 59;
+    private final LocalDateTime attendanceTime;
```

Comment:
상수와 상태는 줄바꿈을 통해서 적절히 분리해주세요. 😃 
```suggestion
    public final static int ABSENT_DEFAULT_MINUTE = 59;
    
    private final LocalDateTime attendanceTime;
```